Sources/MessagingInApp/Gist/GistDelegate.swift

```diff
@@ -41,7 +41,13 @@ class GistDelegateImpl: GistDelegate {
         if let deliveryId = message.campaignDeliveryId {
             eventBusHandler.postEvent(TrackInAppMetricEvent(deliveryID: deliveryId, event: InAppMetric.opened.rawValue))
         }
-
+        // To ensure the keyboard is dismissed on displaying an in-app message,
+        // Update UI on main thread only.
+        // Dismiss keyboard on showing the message to prevent gap between
+        // displaying the in-app message and dismissing the keyboard.
+        DispatchQueue.main.async {
```

Comment:
don't we have a dispatcher util available in this class? if not cant we make it available instead?

Reply:
I think you're referring to `ThreadUtil` class ? 🤔  If yes, then I have implemented that in the latest commit.